it/uk/gov/hmrc/helptosave/support/FakeRelationshipService.scala

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmrc.helptosave.support
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration._
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Suite}
+
+trait FakeRelationshipService extends BeforeAndAfterAll with BeforeAndAfterEach with ScalaFutures {
```

Comment:
Also, self-typing as `BeforeAndAfterAll`, `BeforeAndAfterEach` and `ScalaFutures` would probably make it clearer what exactly is being brought in the extending class. And this trait extending `ScalaFutures` doesn't make sense here anyway since it is not being used, so you wouldn't need to self type as `ScalaFutures` if you were to do self-typing here.